libraries/core-react/src/Slider/MinMax.tsx

```diff
@@ -23,12 +23,12 @@ const StyledMinMax = styled.span`
   }
 `
 
-type Props = {
+type MinMaxProps = {
   /** Children is required */
-  children: ReactNode
```

Comment:
Actually I can change back to `ReactNode`, this was a change I did because of this error in `Slider.tsx`. But I fixed it by changing return type from `void` to `number | string` instead. 
![Screenshot 2020-11-03 at 10 29 24](https://user-images.githubusercontent.com/31915755/97968840-7d1ddc00-1dbf-11eb-92c3-7d296d4e5e49.png)